api/utils_test.go

```diff
@@ -174,31 +199,50 @@ func loadTestData(db *gorm.DB) {
 		Path:        "/i/hold/the/universe/on/my/waist",
 	}
 
-	db.Create(&testUser)
-	db.Create(&testAddress)
-
-	firstOrder.ID = "first-order"
-	firstOrder.LineItems = []*models.LineItem{&firstLineItem}
-	firstOrder.CalculateTotal(&calculator.Settings{}, nil)
-	firstOrder.BillingAddress = testAddress
-	firstOrder.ShippingAddress = testAddress
-	firstOrder.User = &testUser
-	db.Create(&firstLineItem)
-	db.Create(firstTransaction)
-	db.Create(firstOrder)
-
 	secondOrder.ID = "second-order"
-	secondOrder.LineItems = []*models.LineItem{&secondLineItem1, &secondLineItem2}
+	secondOrder.LineItems = []*models.LineItem{secondLineItem1, secondLineItem2}
 	secondOrder.CalculateTotal(&calculator.Settings{}, nil)
 	secondOrder.BillingAddress = testAddress
 	secondOrder.ShippingAddress = testAddress
-	secondOrder.User = &testUser
+	secondOrder.User = testUser
+	secondTransaction.ID = "second-trans"
 	secondTransaction.Amount = secondOrder.Total
 	secondTransaction.Status = models.PaidState
-	db.Create(&secondLineItem1)
-	db.Create(&secondLineItem2)
-	db.Create(secondTransaction)
-	db.Create(secondOrder)
+
+	return &TestData{
+		fmt.Sprintf("/users/%s/orders", testUser.ID),
+		fmt.Sprintf("/orders/%s", firstOrder.ID),
+
+		testToken(testUser.ID, testUser.Email),
+		testUser,
+		testAddress,
+
+		firstOrder,
+		firstTransaction,
+		firstLineItem,
+
+		secondOrder,
+		secondTransaction,
+		secondLineItem1,
+		secondLineItem2,
+	}
+}
+
+func loadTestData(db *gorm.DB) *TestData {
+	testData := setupTestData()
+
+	db.Create(testData.testUser)
```

Comment:
What do you think about wrapping these each in a `require.NoError(t, xxxx.Create().Error())`